components/DeleteButton/index.js

```diff
@@ -1,40 +1,37 @@
-import GetNotes from "../GetNotes";
-import { useEffect, useState } from "react";
+import { useState, useEffect } from "react";
+import useStore from "../../src/store/store";
 
-export default function DeleteButton({ itemDate, itemId, notes, setNotes }) {
-  const [deleteThis, setDeleteThis] = useState([]);
-  const [clicked, setClicked] = useState("false");
-
-  function findItemByDate() {
-    const itemToDelete = notes.filter((item) => {
-      return item.date === itemDate;
-    });
-    setDeleteThis(itemToDelete);
-    setClicked("true");
+export default function DeleteButton({ uuid }) {
```

Comment:
Why are you using the `uuid` of the cards instead of the normal given `id`?

Reply:
because there is a high possibilty the user will pick the same card on different days. would be problematic to filter them out by the data.js id